ci2/ci/github.py

```diff
@@ -128,17 +165,78 @@ def from_gh_json(gh_json, target_branch):
 
         review_state = 'pending'
         for login, state in latest_state_by_login.items():
-            if (state == 'CHANGES_REQUESTED'):
+            if state == 'CHANGES_REQUESTED':
                 review_state = 'changes_requested'
                 break
-            elif (state == 'APPROVED'):
+            if state == 'APPROVED':
                 review_state = 'approved'
             else:
-                assert state == 'DISMISSED' or state == 'COMMENTED', state
+                assert state in ('DISMISSED', 'COMMENTED'), state
 
         self.review_state = review_state
 
+    async def start_build(self, batch_client):
+        assert not self.batch
+
+        # FIXME this needs to be per-PR and async
+        try:
+            async with repos_lock:
+                target_repo = self.target_branch.branch.repo
+                repo_dir = f'repos/{target_repo.short_str()}'
+
+                shell('git', 'config', 'user.email', 'hail-ci-leader@example.com')
```

Comment:
the issue with your commits is that you do this outside of a repo so it's a global config

Reply:
We can just do this config once in the CI docker image